modules/standard/Lists.chpl

```diff
@@ -54,13 +54,13 @@
 module Lists {
 
   pragma "no doc"
-  config const _initialCapacity = 8;
+  private const _initialCapacity = 8;
 
   pragma "no doc"
-  config const _initialArrayCapacity = 16;
+  private const _initialArrayCapacity = 16;
 
   pragma "no doc"
-  config param _sanityChecks = true;
+  private config param _sanityChecks = true;
```

Comment:
I know this is not related to this PR but since this is a `config`, should the name be more verbose? Even if it is not a user-facing feature..

Reply:
Thanks for the reminder. Actually, it makes no sense for it to be both private and config, since the concept of a "private config var" is inherently useless...

I'll probably just remove the `config` keyword.